meetings/cpansec-minutes-2024-11-15.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+toc: true
+meeting_time: 2024-11-15 16:00 UTC
+title: Minutes 2024-11-15
+---
+
+## Meeting details
+
+* 2024-11-15 16:08 UTC on #cpansec-discussion on Matrix
+
+## 15:30 UTC – Pre-meeting socializing
+
+*   Socializing & getting up to speed before the meeting starts properly
+*   Discuss organizing projects, swimlanes and issues (...)
+
+## 16:00 UTC – Meeting start
+
+### Welcome
+
+*   @thibaultduponchelle joined :)
+*   Meeting chair: @timlegge
+*   Meeting secretary: @sjn & @tux
+
+### Attendees, absents & regrets
+
+*   Attendees
+    * @sjn, @stigtsp, @tux, @thibaultduponchelle, @timlegge
+*   Regrets
+    * @rrwo
+*   Partly Absent
+    * 
+
+### Approve previous meeting minutes
+
+*   Previous meeting minutes was approved by @sjn, @timlegge, @tux, @garu
+
+### Quick summary of current work (Grouped by project)
+
+1.  [CPAN Metadata & Software Bills of Materials](https://github.com/orgs/CPAN-Security/projects/1)
+    *   stalled
+2.  [CPAN Privacy and Compliance](https://github.com/orgs/CPAN-Security/projects/9)
+    *   stalled
+3.  [CPAN Provenance & Supply Chain Security](https://github.com/orgs/CPAN-Security/projects/3)
+    *   nothing to do
+4.  [CPAN Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+    *   @stigtsp took over the active ticket
+    *   @tux will write some introduction to the use/need of Test::CVE, @timlegge will review
+5.  [CPAN Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11)
+    *   @sjn - using distroprefs as a model for this has come up recently. shall we move forward?
+    *   @thibaultduponchelle will look into distroprefs support for cpanm and friends
+6.  [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   @thibaultduponchelle takes one of the tickets
+7.  [CPAN Transparency Logs](https://github.com/orgs/CPAN-Security/projects/2)
+    *   …
+8.  [CPAN Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+    *   …
+9.  [CPANSec Governance, Policy & Funding](https://github.com/orgs/CPAN-Security/projects/7)
+    *   …
+
+
+
+### TLS/HTTPS/CSPRNG/DSA in core
+*   @rrwo, @stigtsp - CSPRNG and DSA should be in-scope (they suggest and hope)
```

Comment:
```suggestion
*   @robrwo, @stigtsp - CSPRNG and DSA should be in-scope (they suggest and hope)
```
Not sure if this should be attributed to Rob since he wasn't in the meeting? You ok with that @robrwo?